src/content/docs/guides/Web development.mdx

```diff
@@ -128,9 +128,70 @@ Najpopularniejsza obecnie biblioteka (która działa jak framework) do tworzenia
 
 ## Accessibility
 
-Alt na fotkach
+### Opisy obrazków
+
+Dobrą praktyką jest umieszczanie krótkiego opisu obrazka poza samym src obrazka. Służy to czytnikom ekranów, z który korzystają, np. niewidomi oraz pomaga w SEO w Google i innych wyszukiwarkach. Sprawa jest bardzo prosta:
+
+```jsx {8}
+// app/page.tsx
+import Image from 'next/image'
+
+export function MyComponent() {
+  return (
+    <Image
+      src="/next.svg"
+      alt="Next.js logo" // Krótki opis grafiki
+      width={180}
+      height={38}
+      priority
+    />
+  )
+}
+```
+
+### Przyciski jako linki
+
+Za wsadzanie linka w przycisk będę karał tygodniową chłostą, to nie jest poprawny html, sam react krzyczy, że nie można tak robić i czytniki ekranów się popierdolą jak spróbują to przeczytać
+
+Jak zrobić Link, który wygląda jak przycisk: 
+```jsx
+// app/page.tsx
+import Link from "next/link"
+import { Button } from "@/components/ui/button"
+ 
+export function ButtonAsChild() {
+  return (
+    <Button asChild>
+      <Link href="/login">Login</Link>
+    </Button>
+  )
+}
+```
+
+### Kontrola focusu
 
-Przyciski a linki
+Czasami w standardowym HTML po kliknieciu TAB lub SPACJI czy ENTER możemy zaobserwować brzydki, gruby ring dookoła przycisku/elementu. Jest to `outline`, który domyślnie jest włączony ale zgódźmy się, pięknie nie wygląda. Jest też w niektórych komponentach `shadcn` (chodź tu pasuje do przycisków czy inputów, chyba, że robimy wariant ghost przycisku i nie chcemy tego).
```

Comment:
oj oj, to jest bardzo zła praktyka i nie wiem skąd to wziąłeś, to totalnie psuje a11y i ludzie poruszający się klawiatura nie wiedzą gdzie mają kursor 

https://stackoverflow.com/questions/19455499/why-is-outlinenone-not-considered-good-practice